src/stim/mem/simd_bit_table.h

```diff
@@ -79,6 +79,10 @@ struct simd_bit_table {
     inline const simd_bits_range_ref<W> operator[](size_t major_index) const {
         return data.word_range_ref(major_index * num_simd_words_minor, num_simd_words_minor);
     }
+    /// Return the index in data of a row of a W * W bit block with given major and minor index
+    inline const size_t data_index(size_t major_index, size_t minor_index, size_t row) const {
```

Comment:
I don't understand what this method is computing. Part of the reason is that I explicitly avoided ever saying the word "row" or "column" in the code, since those require an ordering convention to turn into actual relevant math, so I have no idea what a row refers to here.

Rename the "row" parameter and add and "Args:" section to this method. Also this needs a way better name. "data index"? Might as well call it "number method", lol.